web-common-marlowe/src/Marlowe/Extended.purs

```diff
@@ -91,6 +91,18 @@ class Template a b where
 class Fillable a b where
   fillTemplate :: b -> a -> a
 
+class HasParties a where
+  getParties :: a -> Set S.Party
+
+instance arrayHasParties :: HasParties a => HasParties (Array a) where
+  getParties as = foldMap (\a -> getParties a) as
```

Comment:
`(\a -> getParties a)` is equivalent to `getParties`

Reply:
👍